working/augmentation-libraries/feature-specification.md

```diff
@@ -1286,13 +960,13 @@ For example, we define the *augmented parameter list* of a non-empty stack,
 _This will usually be exactly the parameter list of the introductory
 declaration, but the ordering of named parameters may differ. This is mostly
 intended as an example, in practice the augmented parameter list can just be
-the parameter list of the introductory declaration, but it’s more
+the parameter list of the introductory declaration, but it's more
 direct and clearly correct to use the actual parameter list of the declaration
 when creating the parameter scope that its body will run in._
 
 Similarly we define the _augmented function type_ of the declaration stack.
 Because of the restrictions we place on augmentations, they will all have the
-same function type as the introductory declaration, but again it’s
+same function type as the introductory declaration, but again it's
 simpler to assign a function type to every declaration.
 
 #### Invocation
```

Comment:
All references to 'augmented' in this section should be removed, and the rules simplified accordingly.

Reply:
Done, though again I'm still not really sure what this section is for. :)